tests/cypress/page-object/digitall.home.page.ts

```diff
@@ -15,9 +21,24 @@ class DigitallHomePage extends BasePage {
             .contains('div[role="row"]', page)
             .rightclick({ force: true })
             .should('have.class', 'context-menu-open')
-        this.getIframeBody().contains('span', 'Edit').click()
+        this.getIframeBody().find(this.elements.editSite).click()
         return editPage
     }
+
+    publishSite() {
+        this.getIframeBody()
+            .contains('div[role="row"]', 'Digitall')
+            .rightclick({ force: true })
```

Comment:
Please don't use "force: true" this sometime cause very flaky behaviours

Reply:
"force:true" removed, ended up being unnecessary with the stabilization around iframes.